src/interaction-handlers/ticket/categorizeCrash.ts

```diff
@@ -26,15 +26,16 @@ export class ButtonHandler extends InteractionHandler {
 
 		switch (crashType) {
 			case 'CP':
-				await interaction.channel.send(`**Click the copy button and paste it here.**\n${plsBePatientTY}`);
+				await interaction.channel.send(`**Click the copy symbol (next to the symbol with an arrow pointing directly up) and paste it here.**\n${plsBePatientTY}`);
 				break;
 			case 'launcher':
-				await interaction.channel.send(`1. Open your SkyClient folder. See this: https://youtu.be/BHIM2htfMk8
+				await interaction.channel.send(`You'll need to send your crashlog. Here's how to do so.
+1. Open your SkyClient folder. See this: https://youtu.be/BHIM2htfMk8
 2.
-**If you see a "View Crash Report" button:**
+**If you saw a "View Crash Report" button in the launcher:**
 Go to the \`crash-reports\` folder and upload the most recent file here.
-**If you don't:**
-Go to the \`logs\` folder and upload the file called \`latest.log\`.
+**If you didn't see the button:**
+Go to the \`logs\` folder and upload the file called \`latest\` or \`latest.log\`.
```

Comment:
Windows and Mac default behavior is to *not* show file extensions. Showing file extensions is considered an advanced setting, so maybe we can just say "upload the log called `latest`". Anyone who turned on file extensions (hopefully a semi-competent user!) will know what this means.

Linux users should be expected to figure this out themselves.

```suggestion
Go to the \`logs\` folder and upload the log called \`latest\`
```

Reply:
I say leave it as it is. Simply because, some people might have turned it on at one point and forgotten. Plus, from some tickets I see, `latest` is shown as `latest.log` but the person asking for help seems to be that they wouldn't be able to connect the dots. I do agree that Linux users should know what this means, however.

Sorry if that was confusing, tl;dr leave it bc we can't hope

Reply:
I would hope that anyone who sees "log called `latest`" would see `latest.log` and realize that is a "log called `latest`". I'm fine with the wording you chose, but I'd personally rather keep it shorter. I'd like to get another opinion on this.

Reply:
Or maybe instead of "log called `latest`", we could say "log labelled latest"? In my mind, I feel like that would make the phrasing slightly more ambiguous, leading the user to not take the instructions quite so literally.
```suggestion
Go to the \`logs\` folder and upload the log labelled "latest".
```